.github/pull_request_template.md

```diff
@@ -1,25 +1,28 @@
 TODO: Please describe your changes here.
 
-TODO: Delete the following lines but _keep and fill out_ (!) the checklist below.
+<!--
```

Comment:
FYI: When we had the instructions in comments before, a lot of people didn't delete them. I found that annoying, especially when helping update PR descriptions and having to scroll past obsolete stuff. See https://github.com/unicode-org/icu/pull/3213